File: tests/judge/malicious-programs/infinite_loop.c
/* 恶意样例：无限循环（墙钟/CPU 时间超限）。
 * 预期沙箱结果：SIGKILL 超时终止，映射 TLE；不占用宿主机资源。 */
int main(void) {
    while (1) {
    }
    return 0;
}

File: tests/judge/malicious-programs/InfiniteLoop.java
# 恶意样例：超时循环（Java 变体）。
# 预期沙箱结果：墙钟超时，TLE。
public class InfiniteLoop {
    public static void main(String[] args) {
        while (true) {
            Math.sqrt(System.nanoTime());
        }
    }
}
